learn-responsive-web-design-by-building-a-piano/index09.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Responsive Piano</title>
+</head>
+
+<style>
+
+    html {
+        box-sizing: border-box;
+    }
+
+
+    *, *:before, *:after {
+        box-sizing: inherit;
+    }
+
+</style>
+
+<body>
+
+    <div id="piano">
+
+        <div class="keys">
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+            <div class="key black--key"></div>
+        </div>
+    </div>
+ 
+</body>
+</html>
+
+<!-- 
+    Let's start style and giving form to the piano. Add the id
```

Comment:
Break this down into at least four steps. Have the student create the CSS rule and set the width, then the height, margin, and background-color in the following lessons.